### PR TITLE
M3 1862

### PR DESCRIPTION
**Pull request by nwisnewski**
**Purpose**
Addressed test failures

**To Test**
`yarn && yarn start` new shell `yarn selenium` new shell `yarn e2e:modified && yarn --spec=e2e/specs/domains/smoke-list-domains.spec.js`

### CODE REVIEW

File: e2e/pageobjects/page.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default class Page {
 
     // Breadcrumb Component
     get breadcrumbEditableText() { return $('[data-qa-editable-text]'); }
-    get breadcrumbStaticText() { return $('[data-qa-static-text]'); }
+    get breadcrumbStaticText() { return $('[data-qa-label-title]'); }
```
Comment (patthiel) [inline e2e/pageobjects/page.js]: This was intended to be the label for the static text breadcrumb component here:

http://localhost:6006/iframe.html?selectedKind=Breadcrumb&selectedStory=Static%20text&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

It doesn't seem that `data-qa-label-title` as a selector in that story

**Comment by patthiel** [comment]
Just one note of feedback, otherwise looks good!

File: e2e/pageobjects/page.js (hunk 1 above)
Comment (patthiel) [reply e2e/pageobjects/page.js]: ah 👍  hadn't seen that

**Comment by nwisnewski** [comment]
M3-1862: TEST: e2e suite suite repairs.

**Pull request merged**